spicy/toolchain/src/compiler/codegen/parser-builder.cc

```diff
@@ -1908,6 +1905,10 @@ struct ProductionVisitor : public production::Visitor {
         pushBuilder(builder_alt2);
         parseProduction(*p->alternatives().second);
         popBuilder();
+
+        pushBuilder(builder_default);
+        pb->parseError("no expected look-ahead token found", p->location());
+        popBuilder();
```

Comment:
Nit: this was just moved here, but maybe

```suggestion
        pushBuilder(builder_default, [&]() { pb->parseError("no expected look-ahead token found", p->location()); });
```